app/controllers/api/openid_connect/authorizations_controller.rb

```diff
@@ -0,0 +1,95 @@
+module Api
+  module OpenidConnect
+    class AuthorizationsController < ApplicationController
+      rescue_from Rack::OAuth2::Server::Authorize::BadRequest do |e|
+        logger.info e.backtrace[0, 10].join("\n")
+        render json: {error: e.message || :error, status: e.status}
+      end
+
+      before_action :authenticate_user!
+
+      def new
+        request_authorization_consent_form
+      end
+
+      def create
+        restore_request_parameters
+        process_authorization_consent(params[:approve])
+      end
+
+      private
+
+      def request_authorization_consent_form # TODO: Add support for prompt params
+        if Api::OpenidConnect::Authorization.find_by_client_id_and_user(params[:client_id], current_user)
+          process_authorization_consent("true")
+        else
+          endpoint = Api::OpenidConnect::AuthorizationPoint::EndpointStartPoint.new(current_user)
+          handle_start_point_response(endpoint)
+        end
+      end
+
+      def handle_start_point_response(endpoint)
+        _status, header, response = *endpoint.call(request.env)
+        if response.redirect?
+          redirect_to header["Location"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.